pre-docs/applying_manifest_blocks.md

```diff
@@ -1,6 +1,6 @@
 # Applying Manifest Blocks
 
-Within a plan, you can use Bolt to apply blocks of Puppet code (manifest blocks) to remote nodes. Similar to the `puppet apply` command, which applies a standalone Puppet manifest to a local system, the Bolt `apply` command leverages manifest blocks to pass code to remote nodes from the command line. You can create manifest blocks that use existing content from the Forge, or mix declarative resource configuration via manifest blocks with procedural orchestration and action in a plan. Most features of the Puppet language are available in a manifest block: classes, custom resource types, and functions. Exceptions are noted in [caveats](#caveats). 
+Within a plan, you can use Bolt to apply blocks of Puppet code (manifest blocks) to remote nodes. Similar to the `puppet apply` command, which applies a standalone Puppet manifest to a local system, the Bolt `apply` command leverages manifest blocks to pass code to remote nodes from the command line. You can create manifest blocks that use existing content from the Forge, or mix declarative resource configuration via manifest blocks with procedural orchestration and action in a plan. Most features of the Puppet language are available in a manifest block: classes, custom resource types, and functions. Exceptions are noted in [caveats](#caveats).
```

Comment:
s/Bolt `apply` command/Bolt `apply` statement/ -- to avoid confusion with command line command

Reply:
I'd like to bikeshed a bit on what we call the apply statement in docs. Command definitely seems wrong.

function?
statement?
maneuver?

Reply:
action